packages/block-editor/src/hooks/use-zoom-out.js

```diff
@@ -49,5 +49,5 @@ export function useZoomOut( zoomOut = true ) {
 		) {
 			__unstableSetEditorMode( originalEditingMode.current );
 		}
-	}, [ __unstableSetEditorMode, zoomOut, mode ] );
+	}, [ __unstableSetEditorMode, zoomOut ] );
```

Comment:
Needed to stop zoom out from being triggered again.

Reply:
We can't omit dependencies from this array as it can cause unexpected side effects.

Reply:
Can you explain the conditions you were testing under that "triggered [zoom out] again"?

I've refactored the code so that the `mode` is not a required dependency but I'd like to verify this behaves as intended.